### PR TITLE
[번역]:MiDaS 모델

### PR DESCRIPTION
**Pull request by Lee-JaeWon**
## 라이선스 동의
_변경해주시는 내용에 BSD 3항 라이선스가 적용됨을 동의해주셔야 합니다._<br />
_더 자세한 내용은 [기여하기 문서](https://github.com/PyTorchKorea/hub-kr/blob/master/CONTRIBUTING.md)를 참고해주세요._<br />
_동의하시면 아래 `[ ]`를 `[x]`로 만들어주세요._<br />

- [x] 기여하기 문서를 확인하였으며, 본 PR 내용에 BSD 3항 라이선스가 적용됨에 동의합니다.

## 관련 이슈 번호
_이 Pull Request와 관련있는 이슈 번호를 적어주세요._<br />
_이슈 또는 PR 번호 앞에 #을 붙이시면 제목을 바로 확인하실 수 있습니다. (예. #999 )_

- **#72**: #(숫자)

## PR 종류
_이 PR에 해당되는 종류 앞의 `[ ]`을 `[x]`로 변경해주세요._<br />
- [ ] 오탈자를 수정하거나 번역을 개선하는 기여
- [x] 번역되지 않은 모델 소개를 번역하는 기여
- [ ] 공식 허브의 내용을 반영하는 기여
- [ ] 위 종류에 포함되지 않는 기여

## PR 설명
_이 PR로 무엇이 달라지는지 대략적으로 알려주세요._
모델 번역.

### CODE REVIEW

**Comment by KyubumShin** [comment]
전체적으로 깔끔하게 번역된 것 같습니다! 👍

File: intelisl_midas_v2.md [hunk 1]
```diff
@@ -16,21 +16,20 @@ accelerator: cuda-optional
...
 
-[MiDaS](https://arxiv.org/abs/1907.01341) computes relative inverse depth from a single image. The repository provides multiple models that cover different use cases ranging from a small, high-speed model to a very large model that provide the highest accuracy. The models have been trained on 10 distinct datasets using
-multi-objective optimization to ensure high quality on a wide range of inputs.
+[MiDaS](https://arxiv.org/abs/1907.01341)는 단일 이미지로부터 상대적 역 깊이(relative inverse depth)를 계산합니다. 본 저장소는 작지만 고속의 모델부터 가장 높은 정확도를 제공하는 매우 큰 모델까지 다양한 사례를 다루는 여러 모델을 제공합니다. 또한 모델은 광범위한 입력에서 높은 품질을 보장하기 위해 다목적(multi-objective) 최적화를 사용해 10개의 개별 데이터 셋에 대해 훈련되었습니다. 
 
-### Dependencies
+### 종속 패키지 설치
 
-MiDaS depends on [timm](https://github.com/rwightman/pytorch-image-models). Install with
+MiDas 모델은 [timm](https://github.com/rwightman/pytorch-image-models)을 사용합니다 아래 명령어를 통해 설치해 주세요
```
Comment (FollowerOfScriabin) [inline intelisl_midas_v2.md:25]: 이 문장을 포함해 전반적으로 마침표가 찍히지 않은 문장이 많습니다.

File: intelisl_midas_v2.md [hunk 2]
```diff
@@ -42,23 +41,23 @@ url, filename = ("https://github.com/pytorch/hub/raw/master/images/dog.jpg", "do
...
+모델을 로드합니다. (개요는 [https://github.com/intel-isl/MiDaS/#Accuracy](https://github.com/intel-isl/MiDaS/#Accuracy) 를 참조하세요)
 ```python
 model_type = "DPT_Large"     # MiDaS v3 - Large     (highest accuracy, slowest inference speed)
 #model_type = "DPT_Hybrid"   # MiDaS v3 - Hybrid    (medium accuracy, medium inference speed)
 #model_type = "MiDaS_small"  # MiDaS v2.1 - Small   (lowest accuracy, highest inference speed)
 
 midas = torch.hub.load("intel-isl/MiDaS", model_type)
 ```
-Move model to GPU if available
+가능한 환경이라면, 모델에 GPU를 사용합니다
```
Comment (FollowerOfScriabin) [inline intelisl_midas_v2.md:52]: 'GPU를 사용할 수 있다면, 모델을 GPU 옮깁니다.' 로 바꾸길 제안드립니다.

File: intelisl_midas_v2.md (hunk 1 above)
Comment (Lee-JaeWon) [reply intelisl_midas_v2.md]: 코멘트 감사드립니다. 반영하였습니다!

File: intelisl_midas_v2.md [hunk 3]
```diff
@@ -42,23 +41,23 @@ url, filename = ("https://github.com/pytorch/hub/raw/master/images/dog.jpg", "do
...
+가능한 환경이라면, 모델에 GPU를 사용합니다.
 ```python
 device = torch.device("cuda") if torch.cuda.is_available() else torch.device("cpu")
 midas.to(device)
 midas.eval()
 ```
 
 
-Load transforms to resize and normalize the image for large or small model
+크거나 작은 모델을 위한 이미지를 크기 변경이나 정규화하기 위해 변환(transform)을 로드합니다.
```
Comment (FollowerOfScriabin) [inline intelisl_midas_v2.md:60]: '다양한 크기의 모델에 집어 넣을 입력 이미지의 크기 변경(resize)과 정규화(normalizae)를 위해 변환(transform)을 불러옵니다.' 로 제안드립니다.

File: intelisl_midas_v2.md (hunk 2 above)
Comment (Lee-JaeWon) [reply intelisl_midas_v2.md]: 좋은 것 같습니다. 수정하여 반영하도록 하겠습니다.

File: intelisl_midas_v2.md (hunk 3 above)
Comment (Lee-JaeWon) [reply intelisl_midas_v2.md:60]: 내용 반영해보도록 하겠습니다!

**Review by hyoyoung — APPROVED** [review]
good
